1-basic-react/package.json

```diff
@@ -16,6 +16,7 @@
     "react-dom": "^0.14.6",
     "webpack": "^1.12.9",
     "webpack-dev-server": "^1.14.1"
+    "babel-core": "^6.17.0",
```

Comment:
@saaqibz can you move this up into alphabetical position?  That will also keep the package.json from breaking, since this line has a comma and the webpack-dev-server line doesn't.

Thanks!